src/Directory.Build.props

```diff
@@ -1,6 +1,6 @@
 <Project>
   <PropertyGroup>
-    <Version>1.0.19</Version>
+    <Version>1.0.20</Version>
```

Comment:
wait is this correct? don't we normally increment on separate so we know how to track ver bumps?

Reply:
It's just to avoid an extra PR just to bump the version. Don't need a separate commit for it, the repo gets tagged when we release.